### DISCUSSION TITLE
Logging out user in case of Inactivity

### DISCUSSION THREAD

**Discussion by abhishektalwar** [netbox-community/netbox#6870] [Q&A] [answered]
Hi Everyone,

I recently changed the LOGIN_TIMEOUT to 15 minutes (900 seconds) for my netbox setup. However, it does seem the logout is 15 minutes from login instead of 15 minutes of inactivity.

Is there a way/mechanism to set this so that it is 15 minutes of inactivity?

**Comment by candlerb** [comment, marked as answer]
Duplicate of #6868
